app/controllers/api/v0/reactions_controller.rb

```diff
@@ -8,7 +8,7 @@ def create
           render json: { message: "invalid_user" }, status: :unprocessable_entity
           return
         end
-        Rails.cache.delete "count_for_reactable-#{params[:reactable_type]}-#{params[:reactable_id]}"
+        RedisRailsCache.delete "count_for_reactable-#{params[:reactable_type]}-#{params[:reactable_id]}"
```

Comment:
I was a little worried that this might point to other reactable types but it looks like so far the only key we set is for Article and if we added any from here on out they would point at Redis so this should be good.